apps/emqx_ds_backends/test/emqx_ds_backends_SUITE.erl

```diff
@@ -649,57 +626,20 @@ t_make_iterator_stale_stream(Config) ->
     StartTime = 0,
     Msgs0 = [
         message(<<"foo/bar">>, <<"1">>, 0),
-        message(<<"foo/baz">>, <<"2">>, 1)
+        message(<<"foo/bar">>, <<"2">>, 1)
     ],
     ?assertMatch(ok, emqx_ds:store_batch(DB, Msgs0)),
+    timer:sleep(1_000),
 
     [{_, Stream0}] = emqx_ds:get_streams(DB, TopicFilter, StartTime),
 
     ok = emqx_ds:add_generation(DB),
     ok = emqx_ds:drop_generation(DB, GenId0),
+    timer:sleep(1_000),
 
     ?assertEqual(
         {error, unrecoverable, generation_not_found},
         emqx_ds:make_iterator(DB, Stream0, TopicFilter, StartTime)
-    ),
-
-    ok.
-
-t_get_streams_concurrently_with_drop_generation(Config) ->
```

Comment:
This testcase was too backend-specific. This test suite is meant to verify properties common for all backends.